cdc/sorter/leveldb/leveldb.go

```diff
@@ -229,6 +231,20 @@ func (ldb *DBActor) Poll(ctx context.Context, tasks []actormsg.Message) bool {
 				log.Panic("db error", zap.Error(err))
 			}
 		}
+		if task.DeleteReq != nil {
+			ldb.deleteCount += task.DeleteReq.Count
```

Comment:
for DBActor, task.DeleteReq.Count  is always 0?

Reply:
For now, it's always zero, we will fill `Count` in later PRs.